__tests__/app.test.js

```diff
@@ -52,9 +52,59 @@ describe('/api' , () => {
             .get('/api')
             .expect(200)
             .then(({body: {endpoints}}) => {
-                console.log(endpoints)
                 expect(endpoints).toEqual(endpointsObj)
             })
         })
     })
+})
+
+describe('/api/articles/:article_id' , () => {
+    describe('GET' , () => {
+        test('GET: 200 - returns with the specified article information from the specified id' , () => {
+            return request(app)
+            .get('/api/articles/1')
+            .expect(200)
+            .then(({body: {article}}) => {
+                expect(article.author).toBe("butter_bridge")
+                expect(article.title).toBe("Living in the shadow of a great man")
+                expect(article.article_id).toBe(1)
+                expect(article.body).toBe("I find this existence challenging")
+                expect(article.topic).toBe("mitch")
+                expect(article.created_at).toBe("2020-07-09T20:11:00.000Z")
+                expect(article.votes).toBe(100)
+                expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+            })
+        })
+        test('GET: 200 - returns with the specified article information from the specified id' , () => {
+            return request(app)
+            .get('/api/articles/5')
+            .expect(200)
+            .then(({body: {article}}) => {
+                expect(article.author).toBe("rogersop")
+                expect(article.title).toBe("UNCOVERED: catspiracy to bring down democracy")
+                expect(article.article_id).toBe(5)
+                expect(article.body).toBe("Bastet walks amongst us, and the cats are taking arms!")
+                expect(article.topic).toBe("cats")
+                expect(article.created_at).toBe("2020-08-03T13:14:00.000Z")
+                expect(article.votes).toBe(0)
+                expect(article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
+            })
```

Comment:
There's no need for the second test with a different id. I mean you can keep it, but there's no need for it 🙂